### ISSUE TITLE
[Bug] Guest network switch does not work

### ISSUE THREAD

**Issue by loscalino** [Vaskivskyi/ha-asusrouter#727]
### The problem

The Guest network control switch no longer works, throws an error in the logs and on the GUI just returns to whatever position it was last in.

### Your device model

RT-AX58U

### Firmware type

Merlin

### Firmware version

3004.388.4

### Integration version with the issue

0.25.0b0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2023.11.3

### Is there anything useful in the logs?

```txt
This error originated from a custom integration.
Logger: custom_components.asusrouter.entity
Source: custom_components/asusrouter/entity.py:168
Integration: AsusRouter ([documentation](https://asusrouter.vaskivskyi.com/), [issues](https://github.com/Vaskivskyi/ha-asusrouter/issues))
First occurred: 11:06:27 AM (1 occurrences)
Last logged: 11:06:27 AM

Unable to set state with an exception: set_state() takes from 2 to 4 positional arguments but 5 were given
```

### Additional information

Guest networks work as expected from web admin interface.

**Comment by Vaskivskyi** [comment]
Hello. I am sorry that you are experiencing this issue. It will be fixed in the next release

**Comment by Vaskivskyi** [comment]
Released in `0.25.0`
